app/controllers/idv/otp_delivery_method_controller.rb

```diff
@@ -25,6 +26,12 @@ def create
 
     private
 
+    attr_reader :idv_phone
+
+    def phone_number_capabilities
+      @phone_number_capabilities ||= PhoneNumberCapabilities.new(idv_phone, phone_confirmed: false)
```

Comment:
We should probably do a real check of `phone_confirmed` against user confirmed phone numbers here, in case we'd support voice for confirmed numbers.

Reply:
Updated in 035f1dc